### PR TITLE
Angular17 hanwen

### PR DESCRIPTION
**Pull request by hcaij3**

### CODE REVIEW

File: projects/mf-rt/src/app/container/container.component.html
Comment (linghanHC) [inline projects/mf-rt/src/app/container/container.component.html:1]: you can reference md-rt for container component which has lazy loading of formbase. 
mf only has one site (no need for "internal" variable)

File: projects/mf-rt/src/app/form-base/form-base.component.ts
Comment (linghanHC) [inline projects/mf-rt/src/app/form-base/form-base.component.ts:1]: remove isInternal field, no need for it

File: projects/mf-rt/src/app/global/global.service.ts
Comment (linghanHC) [inline projects/mf-rt/src/app/global/global.service.ts:1]: remove isInternal field, no need for it

File: projects/mf-rt/src/app/models/transaction.ts
Comment (linghanHC) [inline projects/mf-rt/src/app/models/transaction.ts:1]: use the common entities eg. (IText , IIdText, IIdTextLabel, ITextLabel  .....) from the lib proj

File: projects/mf-rt/src/environments/environment.prod.ts
Comment (linghanHC) [inline projects/mf-rt/src/environments/environment.prod.ts:1]: this is not needed as we already have prod-en and prod-fr

File: projects/mf-rt/src/environments/environment.ts
Comment (linghanHC) [inline projects/mf-rt/src/environments/environment.ts:1]: remove isInternal

File: projects/mf-rt/src/app/container/container.component.html
Comment (hcaij3) [reply projects/mf-rt/src/app/container/container.component.html:1]: changed

File: projects/mf-rt/src/app/form-base/form-base.component.ts
Comment (hcaij3) [reply projects/mf-rt/src/app/form-base/form-base.component.ts:1]: removed

File: projects/mf-rt/src/app/global/global.service.ts
Comment (hcaij3) [reply projects/mf-rt/src/app/global/global.service.ts:1]: removed

File: projects/mf-rt/src/app/models/transaction.ts
Comment (hcaij3) [reply projects/mf-rt/src/app/models/transaction.ts:1]: changed

File: projects/mf-rt/src/environments/environment.prod.ts
Comment (hcaij3) [reply projects/mf-rt/src/environments/environment.prod.ts:1]: removed

File: projects/mf-rt/src/environments/environment.ts
Comment (hcaij3) [reply projects/mf-rt/src/environments/environment.ts:1]: removed

**Review by linghanHC — APPROVED** [review]

**Pull request merged**